GhSA/Helpers/PostHog.cs

```diff
@@ -7,25 +7,70 @@
 using System.Threading.Tasks;
 using Grasshopper.Kernel;
 using Newtonsoft.Json;
+using static GsaGH.Helpers.PostHogUser;
 
 namespace GsaGH.Helpers
 {
+  public static class PostHogUser
+  {
+    public class User
```

Comment:
if it´s public it deserves its own file.. ;) 
I guess it can be private though?